gulpfile.js

```diff
@@ -14,5 +14,6 @@ var elixir = require('laravel-elixir');
 elixir(function(mix) {
     mix.copy('node_modules/@dosomething/forge/dist', 'public/assets/vendor/forge');
     mix.copy('node_modules/dosomething-modal/dist', 'public/assets/vendor/modal');
+    mix.browserify('split.js', 'public/assets/js/split.js');
```

Comment:
@deadlybutter I can show you more on this when I'm in tomorrow. We'll defs be adding more scripts in the near future, so having them bundled all together as a `main.js` like @DFurnes mentions is ideal!  💃